webapp/shop/advantage/views.py

```diff
@@ -89,6 +90,16 @@ def advantage_view(advantage_mapper, **kwargs):
 def get_user_subscriptions(advantage_mapper, **kwargs):
     email = kwargs.get("email")
     user_subscriptions = advantage_mapper.get_user_subscriptions(email)
+    launchpad = Launchpad.login_anonymously(
+        "ubuntu.com/pro", "production", version="devel"
+    )
+    sso_user = user_info(flask.session)
+    lp_user = launchpad.people.getByEmail(email=sso_user["email"])
+    community_members = launchpad.people("ubuntumembers").members
+    if lp_user in community_members:
+        for subscription in user_subscriptions:
+            if subscription.type == "free":
+                subscription.number_of_machines = 50
```

Comment:
Solution seems to work. However, please change it a bit. Don't override the `subscription` here.

Instead please save `lp_user in community_members` as a variable and send it to the `advantage_mapper.get_user_subscriptions`. Please override the number of machines at the end of that function. Still not ideal, but it's a temporary thing anyway.

`user_subscriptions = advantage_mapper.get_user_subscriptions(email)` must return the final shape of the subscriptions.